packages/url_launcher/url_launcher_web/lib/navigator.dart

```diff
@@ -0,0 +1,15 @@
+@JS()
+library navigator.dart;
```

Comment:
You can call this library 'navigator' so you don't have to rename it when you import it later.

Also, please move this file inside `lib/src`, so it can't be accessed by external users of the package.

Reply:
Done